test/libweb3core/crypto.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+	This file is part of cpp-ethereum.
```

Comment:
this are the tests for libweb3core that have dependencies on libethereum. thats why it is in libethereum.